### PR TITLE
10480 change result amount logic

### PR DESCRIPTION
**Pull request by schultztimothy**
##### Description

These updates bring back totals for all of the different models that are being searched so that pagination is accurate for each tab. It adds another query to bring back results for each tab individually. 

Pagination was updated on the fe to request results based on the active tab. 

There is also a new environment variable that sets the elastic index once it is created. It corresponds to {index-name} passed to the above command and is named `ACTIVE_ELASTIC_INDEX`

##### Refers/Fixes

fixes: #10478 and #10480

##### Testing

https://user-images.githubusercontent.com/6887938/166337104-0d3f8029-950b-482a-bdcb-866a41a55f06.mov

### CODE REVIEW

**Comment by schultztimothy** [comment]
I added don't merge until this has been merged and the command has been ran: https://github.com/gitcoinco/web/pull/10554. We can then redirect searches to the new index

File: app/search/models.py
Comment (schultztimothy) [reply app/search/models.py]: 😨 ooops thanks for catching that 😆

**Review by thelostone-mc — APPROVED** [review]

File: app/app/settings.py
Comment (thelostone-mc) [inline app/app/settings.py:900]:
```suggestion
ACTIVE_ELASTIC_INDEX = env('ACTIVE_ELASTIC_INDEX', default='search-index-1')
```

**Pull request merged**
